src/components/Shared/UserSearchField.tsx

```diff
@@ -17,6 +17,7 @@ interface IExternalProps {
   name?: string;
   onBlur?: (touched: boolean) => any;
   onChange?: (newValue: string) => any;
+  beneficiary: string;
```

Comment:
perhaps no name this "beneficiary", but rather "defaultValue" (as the usersarchfield is not always used for "beneificiaries")

Reply:
Good idea thanks :)

Reply:
Done.